app/src/main/java/org/jellyfin/androidtv/ui/preference/dsl/OptionsCategory.kt

```diff
@@ -19,6 +19,16 @@ class OptionsCategory(
 		nodes.add(item)
 	}
 
+	@OptionsDSL
+	fun link(init: OptionsLink.() -> Unit) {
+		this += OptionsLink(context).apply { init() }
+	}
+
+	@OptionsDSL
+	fun action(init: OptionsAction.() -> Unit) {
+		this += OptionsAction(context).apply { init() }
+	}
+
```

Comment:
These should be extension functions in their own files (OptionsLink.kt,OptionsAction.kt). Adding _all_ functions in this file will make the code too long.